_assets/js/utils/debounce.js

```diff
@@ -0,0 +1,13 @@
+// Adapted from Josh W. Comeau: https://www.joshwcomeau.com/snippets/javascript/debounce/
+
+const debounce = (callback, wait) => {
+    let timeoutId = null;
```

Comment:
do you need this variable?

Reply:
Yes, because it used to establish a timeout before firing the callback. See line 7.